### PR TITLE
Update nightly.steps.yaml

### PR DESCRIPTION
**Pull request by simonsarginson**

### CODE REVIEW

File: .buildkite/nightly.steps.yaml [hunk 1]
```diff
@@ -51,10 +51,10 @@ steps:
       message: "exampleproject-4.26 ${BUILDKITE_MESSAGE}"
       env: 
         BUILD_TYPE: "EXAMPLE"
-        GDK_BRANCH: "master"
-        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK"
+        GDK_BRANCH: "0.13.0-rc"
+        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK-0.13.0-rc"
```
Comment (simonsarginson) [inline .buildkite/nightly.steps.yaml:55]:
```suggestion
        GDK_BRANCH: "0.13.0-rc" " # NOTE: temp fix for new ci release interop issues
        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK-0.13.0-rc" " # NOTE: temp fix for new ci release interop issues
```

File: .buildkite/nightly.steps.yaml [hunk 2]
```diff
@@ -51,10 +51,10 @@ steps:
...
         ENGINE_MAJOR: "4.26"
-        PROJECT_BRANCH: "${BUILDKITE_BRANCH}"
+        PROJECT_BRANCH: "0.13.0-rc"
```
Comment (simonsarginson) [inline .buildkite/nightly.steps.yaml:57]:
```suggestion
        PROJECT_BRANCH: "0.13.0-rc" " # NOTE: temp fix for new ci release interop issues
```

File: .buildkite/nightly.steps.yaml [hunk 3]
```diff
@@ -54,7 +54,7 @@ steps:
         GDK_BRANCH: "0.13.0-rc" " # NOTE: temp fix for new ci release interop issues
         ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK-0.13.0-rc" " # NOTE: temp fix for new ci release interop issues
         ENGINE_MAJOR: "4.26"
-        PROJECT_BRANCH: "0.13.0-rc"
+        PROJECT_BRANCH: "0.13.0-rc" " # NOTE: temp fix for new ci release interop issues
```
Comment (mironec) [inline .buildkite/nightly.steps.yaml:57]:
```suggestion
        PROJECT_BRANCH: "0.13.0-rc" # NOTE: temp fix for new ci release interop issues
```

File: .buildkite/nightly.steps.yaml [hunk 4]
```diff
@@ -51,10 +51,10 @@ steps:
       message: "exampleproject-4.26 ${BUILDKITE_MESSAGE}"
       env: 
         BUILD_TYPE: "EXAMPLE"
-        GDK_BRANCH: "master"
-        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK"
+        GDK_BRANCH: "0.13.0-rc" " # NOTE: temp fix for new ci release interop issues
+        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK-0.13.0-rc" " # NOTE: temp fix for new ci release interop issues
```
Comment (mironec) [inline .buildkite/nightly.steps.yaml:55]:
```suggestion
        GDK_BRANCH: "0.13.0-rc" # NOTE: temp fix for new ci release interop issues
        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK-0.13.0-rc" # NOTE: temp fix for new ci release interop issues
```

Comment (simonsarginson) [inline .buildkite/nightly.steps.yaml:55]:
```suggestion
        GDK_BRANCH: "0.13.0-rc" # NOTE: temp fix for new ci release interop issues
        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK-0.13.0-rc" # NOTE: temp fix for new ci release interop issues
```

File: .buildkite/nightly.steps.yaml [hunk 5]
```diff
@@ -51,10 +51,10 @@ steps:
       message: "exampleproject-4.26 ${BUILDKITE_MESSAGE}"
       env: 
         BUILD_TYPE: "EXAMPLE"
-        GDK_BRANCH: "master"
-        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK"
+        GDK_BRANCH: "0.13.0-rc" # NOTE: temp fix for new ci release interop issues
+        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK-0.13.0-rc" # NOTE: temp fix for new ci release interop issues
         ENGINE_MAJOR: "4.26"
-        PROJECT_BRANCH: "${BUILDKITE_BRANCH}"
+        PROJECT_BRANCH: "0.13.0-rc" # NOTE: temp fix for new ci release interop issues
```
Comment (mironec) [inline .buildkite/nightly.steps.yaml:57]:
```suggestion
        PROJECT_BRANCH: "${BUILDKITE_BRANCH}"
```
Similarly in the other PRs as well.

File: .buildkite/nightly.steps.yaml [hunk 6]
```diff
@@ -51,10 +51,10 @@ steps:
       message: "exampleproject-4.26 ${BUILDKITE_MESSAGE}"
       env: 
         BUILD_TYPE: "EXAMPLE"
-        GDK_BRANCH: "master"
-        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK"
+        GDK_BRANCH: "0.13.0-rc" # NOTE: temp fix for new ci release interop issues
```
Comment (joshuahuburn) [inline .buildkite/nightly.steps.yaml:54]:
```suggestion
        GDK_BRANCH: "${GDK_BRANCH:-0.13.0-rc}" # NOTE: temp fix for new ci release interop issues
```

File: .buildkite/nightly.steps.yaml [hunk 7]
```diff
@@ -51,10 +51,10 @@ steps:
...
+        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK-0.13.0-rc" # NOTE: temp fix for new ci release interop issues
```
Comment (joshuahuburn) [inline .buildkite/nightly.steps.yaml:55]:
```suggestion
        ENGINE_BRANCH: "${ENGINE_BRANCH:-4.26-SpatialOSUnrealGDK-0.13.0-rc}" # NOTE: temp fix for new ci release interop issues
```

**Review by joshuahuburn — APPROVED** [review]

**Review by mironec — APPROVED** [review]

**Pull request merged**